recording/channels.go

```diff
@@ -0,0 +1,65 @@
+package recording
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"google.golang.org/protobuf/proto"
+
+	livekit "github.com/livekit/protocol/proto"
+	"github.com/livekit/protocol/utils"
+)
+
+const (
+	ReservationChannel = "RESERVE_RECORDER"
+	ResultChannel      = "RECORDING_RESULT"
+	RequestTimeout     = time.Second * 3
+	ReservationTimeout = time.Second * 2
+)
+
+func RequestChannel(id string) string {
+	return "RECORDING_REQUEST_" + id
+}
+
+func ResponseChannel(id string) string {
+	return "RECORDING_RESPONSE_" + id
+}
+
+func RecordingRPC(ctx context.Context, mb utils.MessageBus, recordingId string, req *livekit.RecordingRequest) error {
```

Comment:
would you ever return a non-void response?

Reply:
maybe in the future, but for now it's just an error